python/private/common/providers.bzl

```diff
@@ -13,14 +13,15 @@
 # limitations under the License.
 """Providers for Python rules."""
 
-load("@rules_python_internal//:rules_python_config.bzl", "config")
+load("//python/private:util.bzl", "IS_BAZEL_6_OR_HIGHER")
```

Comment:
Nit, what about using bazel_features package for this? I know that this adds an extra dep, but we are already using that on with bzlmod setups.